components/canvas/SubActivityNode.tsx

```diff
@@ -143,13 +143,13 @@ export const SubActivityNode = ({
           description={description}
         />
         <div className={styles["node__role-container"]}>
-          <Typography variant="caption">
-            {formatNodeText(role ?? "", 16, true)}
+          <Typography variant="caption" className={styles["node__info-text"]}>
+            {formatNodeText(role ?? "")}
```

Comment:
It's a little messy having truncation in both the CSS and `formatNodeText`. I think truncation should be handled by CSS.

Reply:
The truncation is handled by CSS now. I can delete the truncation part of formatNodeText.